scripts/individual-tests.sh

```diff
@@ -6,15 +6,21 @@ set -u -e
 
 FAILURES=""
 
+# A space-separated list of relative paths to exclude from the test run. Example: "./Blocks/HTTP ./Blocks/MMQ".
+EXCLUDE=("")
+
 for i in $(find . -name test.cc | sort -g) ; do
   DIR=$(dirname $i)
-  echo -e -n "\n\033[0m\033[1mDir\033[0m: \033[36m"
-  echo $DIR
-  cd $DIR
-  if ! make -s test ; then
-    FAILURES="$FAILURES\n- $DIR"
+  echo -e "\n\033[0m\033[1mDir\033[0m: \033[36m${DIR}\033[0m"
+  if [[ " ${EXCLUDE[@]} " =~ " $DIR " ]]; then
```

Comment:
@dkorolev That's not bash-specific. For the edge case of matching the first and the last item: always match an item which has space from both sides.

For example, it was done the same way in JavaScript (jQuery) to find an item in a space-separated className string without splitting and building dynamic regexps before the `classList.has()` API was made available.